api/auth/interfaces/UserInfo.ts

```diff
@@ -0,0 +1,8 @@
+export interface UserInfo {
+  forename?: string
+  surname?: string
+  email?: string
+  active?: boolean
+  id?: string
+  roles?: string[]
+}
```

Comment:
I am guessing there is a valid reason for all properties being nullable? Seems a pretty flakey model